nvOC

```diff
@@ -210,16 +210,35 @@ Reboot()
    sudo su -c "shutdown -r now"
 }
 
+############################### Restart Miner ##################################
+Restart()
+{
+   # Reboots the host
+        Stop() && Start()
+}
 ############################### Start nvOC ####################################
 Start()
 {
-    if [ $LOCALorREMOTE == "LOCAL" ];then
-       gnome-terminal
-    else   
-       Reboot
-    fi   
+   pkill -f gnome-terminal
+   pkill -f 5watchdog
+   pkill -f 7telegram
+   pkill -f 6tempcontrol
+   pkill -e screen
+   pkill -f 8wtm
```

Comment:
Is it needed to kill these before start 3main?
We have all the kill commands for stop

Reply:
And 3main re-up those needed

Reply:
I think just killing gnome-terminal to prevent double instance would be enough